Library/Formula/nload.rb

```diff
@@ -11,6 +11,11 @@ class Nload < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
 
+  # F2 Option Crash and Wrong adapter read, see http://sourceforge.net/p/nload/bugs/8/
```

Comment:
Could you try to email the author about this patch? It worries me that there's been no reply for a year.